### PR TITLE
MCOL-4063 Make scripts Python 2/3 capable.

### PR DESCRIPTION
**Pull request by drrtuy**

### CODE REVIEW

File: oam/install_scripts/mcs-loadbrm.py [hunk 1]
```diff
@@ -1,70 +1,88 @@
...
+        config_root.find('./SystemConfig/DataFilePlugin').text = "libcloudio.so"
+        cs_config.write('/etc/columnstore/Columnstore.xml')
+
+        try:
+            brm_saves_current = subprocess.check_output(['smcat', brm])
         except subprocess.CalledProcessError as e:
-            # will happen when brm file does not exist
-            print(f'{brm} does not exist.', file=sys.stderr)
+                # will happen when brm file does not exist
+                print('{} does not exist.'.format(brm), file=sys.stderr)
```
Comment (jmrojas2332) [inline oam/install_scripts/mcs-loadbrm.py]: python interpreter will have issue with this indentation.

**Review by jmrojas2332 — CHANGES_REQUESTED** [review]
Just one issue

File: oam/install_scripts/mcs-loadbrm.py [hunk 2]
```diff
@@ -1,70 +1,88 @@
-#!/usr/bin/env python3
-
-import configparser
+#!/usr/bin/env python
```
Comment (jmrojas2332) [inline oam/install_scripts/mcs-loadbrm.py]: This shebang causes issue on centos8

Comment (drrtuy) [reply oam/install_scripts/mcs-loadbrm.py]: We now choose the appropriate shebang using the platform we build packages for.

File: oam/install_scripts/mcs-loadbrm.py (hunk 1 above)
Comment (drrtuy) [reply oam/install_scripts/mcs-loadbrm.py]: Fixed this.

File: oam/install_scripts/CMakeLists.txt [hunk 3]
```diff
@@ -23,6 +23,15 @@ configure_file("${CMAKE_CURRENT_SOURCE_DIR}/mcs-loadbrm.service.in" "${CMAKE_CUR
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/mcs-storagemanager.service.in" "${CMAKE_CURRENT_SOURCE_DIR}/mcs-storagemanager.service" @ONLY)
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/mcs-stop-controllernode.sh.in" "${CMAKE_CURRENT_SOURCE_DIR}/mcs-stop-controllernode.sh" @ONLY)
 
+IF(RPM MATCHES "(rhel|centos)8")
+    SET(PYTHON_SHEBANG "#!/usr/bin/env python3")
+ELSE
+    SET(PYTHON_SHEBANG "#!/usr/bin/env python")
+ENDIF()
```
Comment (drrtuy) [inline oam/install_scripts/CMakeLists.txt:30]: Here is a syntax error.

**Review by jmrojas2332 — APPROVED** [review]
LGTM

**Pull request merged**